webviz_subsurface/plugins/_map_viewer_fmu/map_viewer_fmu.py

```diff
@@ -68,9 +71,14 @@ class MapViewerFMU(WebvizPluginABC):
 Well picks can be exported from RMS using this script: [extract_well_picks_from_rms.py]\
 (https://github.com/equinor/webviz-subsurface-testdata/tree/master/observed_data/\
 drogon_well_picks/extract_well_picks_from_rms.py)
+
+See     [this file]\
+https://raw.githubusercontent.com/emerson-eps/color-tables/main/react-app/src/component/\
+color-tables.json for color_tables format.
```

Comment:
Here we could then point to our own JSON-file (which could even be YAML if we think that is easier for users to write.